.gitignore

```diff
@@ -0,0 +1 @@
+*.exe
```

Comment:
Can use this chance to use a gitignore template such as [this](https://github.com/infranyx/go-microservice-template/blob/dev/.gitignore) (or any other).